src/amo/pages/SearchTools/index.js

```diff
@@ -81,7 +77,6 @@ export class SearchToolsBase extends React.Component<InternalProps> {
 
 export function mapStateToProps(state: AppState) {
   const filters = {
-    addonType: ADDON_TYPE_OPENSEARCH,
```

Comment:
I am not sure about this, but I don't know what else we can do here so I filed https://github.com/mozilla/addons-frontend/issues/9248